lib/ledger_sync/ledgers/xero/operation.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Ledgers
+    module Xero
+      class Operation
+        module Mixin
```

Comment:
Is there a reason this is not in Core?

Reply:
Yeah, these are specific stuff for Xero (ie oauth handling, resource endpoint naming). Some of these could be in the main Core Operation mixin